content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -1031,7 +1032,7 @@ A component of the [Filled form](#filled-form) event.
 | `id`                    | yes      | `string`                | Field id, for all field types                                                                            |
 | `label`                 | yes      | `string`                | Field label; for all field types                                                                         |
 | `answer`                | yes      | `any`                   | For all field types                                                                                      |
-| `answer.id`             | yes      | `string`                | Answer id; for all field types                                                                           |
+| `answer.id`             | yes      | `string`                | A unique identifier of each option a Customer can choose. For all field types.                           |
```

Comment:
Everything is perfect, but  I just wondering if `unique` is good here, because `id` is unique only among the answers list for the given question.

Reply:
@Adimus11 So you're saying that this `id` won't be unique when out of context of `answer`? That seems reasonable. So maybe we should simply go for `An identifier...`

Reply:
@ale-kacperczyk Sounds reasonable 👍